documentation/asciidoc/computers/raspberry-pi/introduction.adoc

```diff
@@ -147,13 +147,13 @@ a|
 
 8GB | 40-pin GPIO header
 a|
-* 2 × micro HDMI
-* 2 × USB 2.0
-* 2 × USB 3.0
+* 2× micro HDMI
+* 2× USB 2.0
+* 2× USB 3.0
 * CSI camera port
 * DSI display port
 * 3.5mm AV jack
-* PoE-capable Gigabit Ethernet (1Gb/s)
+* Gigabit (1Gb/s) Ethernet RJ45 with PoE support
```

Comment:
Perhaps this should say PoE+ ? https://www.raspberrypi.com/news/announcing-the-raspberry-pi-poe-hat/
(AFAICT it seems like the Pi itself is either "supports PoE" or "doesn't support PoE", and it's the HAT itself that determines whether that PoE is "normal PoE" or "PoE+" ?)